### PR TITLE
Adding plasma-cash like designs docs

### PR DESCRIPTION
**Pull request by nanspro**
- [x] I have read and understood the [Plasma Group Contributing Guide and Code of Conduct](https://github.com/plasma-group/pigi/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

------------------------------------------------------------------------
Fixes https://github.com/plasma-group/pigi/issues/42, #78 

## Description
Explaining why plasma cash-like designs makes it possible for clients to be lightweight?

### CODE REVIEW

**Comment by nanspro** [comment]
@kfichter  i caught fever yesterday and so am not very well, i still have to change this and other PRs to fit according to the rst conventions and also i have to add the diagrams as well. Everything is already done in google docs but i need to add it here so i'll definitely finish it by tomorrow but today i need to sleep :disappointed:

**Comment by ChrisChinchilla** [comment]
@nanspro I made a review here - https://github.com/aerophile/pigi/pull/3 I also have one comment I'll add

File: packages/docs/src/pg/src/plasma/design.rst [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+Anyone can exit their coin by providing the last two transactions in the coin’s ownership history (ie. the coin they are 
+exiting C and its parent P( C )).
+
+An exit can be challenged in three ways: (i) provide a proof of a transaction spending C, (ii) provide a proof of a 
+transaction spending P( C ) that appears before C, (iii) provide a transaction C* in the coin’s history before P( C )
+
+A challenge of type (i) and (ii) blocks the exit immediately. A challenge of type (iii) can be responded to by providing 
+the direct child of C*, which must be either equal to or before P( C )
+
+Would coin ID => denomination be stored as a mapping on the root chain?
```
Comment (ChrisChinchilla) [inline packages/docs/src/pg/src/plasma/design.rst]: @nanspro These lines here I don't entirely understand, 151 down to 158.

Comment (nanspro) [reply packages/docs/src/pg/src/plasma/design.rst]: yeah they were kind of misplaced and the non inclusion stuff is related to SMT's which has already been explained above so i removed that part.

**Comment by ChrisChinchilla** [comment]
OK, here I'm taking your work for #42 so paying out that bounty and will work with plasma team to get the final polish and the work merged!

**Comment by kfichter** [comment]
Heyo! We're doing some repo overhauls and our docs are moving around. We're going to close this PR until we can find the right place to put this information.

**Pull request closed without merging**